labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -88,7 +79,7 @@ object ListManipulationExercise01 {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList.zipWithIndex.collect{ case (e, i) if ((i + 1) % 2) != 0 => e }
```

Comment:
`(i+1) % 2 != 0` is an interesting way to write `i % 2 == 0`. I think you also had the same misunderstanding that I did - actually, I'm still not sure which is correct - the other interpretation is that it actually wanted *those elements that are odd-valued* and not the 1st, 3rd, .. etc elements.